sim/dex.ts

```diff
@@ -1139,6 +1124,7 @@ export class ModdedDex {
 			}
 
 			// gender
+			const template = dexes['base'].getTemplate(set.species || set.name);
 			if (set.gender && set.gender !== template.gender) {
```

Comment:
```suggestion
			if (set.gender) {
```

So yeah, I agree, let's do this.